dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/CacheConfigManager.java

```diff
@@ -0,0 +1,116 @@
+package io.dockstore.webservice.helpers;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.CacheStats;
+import com.google.common.cache.LoadingCache;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CacheConfigManager {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CacheConfigManager.class);
+
+    private static CacheConfigManager cacheConfigManager = new CacheConfigManager();
+
+    private static LoadingCache<String, String> installationAccessTokenCache;
+
+    private static String jsonWebToken;
```

Comment:
I couldn't get the validation to work to see if a JWT is still valid, so for now it will recreate each time the endpoint is called. I do plan on getting this working, I just thought it was important to get this merged so that others can continue working on services.

For now I'll add the volatile keyword.